### PR TITLE
Small set of changes ahead of official v1.0 release

### PR DESCRIPTION
**Pull request by hemmelig**
## What is the purpose of this Pull Request? 
Introduces Continuous Integration (Travis CI) for the project, which tests the package against Python 3.6, 3.7, and 3.8 as well as against Linux (Xenial) and MacOSX.

Also includes:

- Update to copyright notices (99868e3)
- Pin pandas version to ensure old versions are updated (c78e45e)
- Prep. for pip installation (b2e7391)
- Add badges! Exciting! (194297f

### Final checklist
- [x] All tests still pass.
- [x] Any new features or fixed regressions are covered by new tests.

### CODE REVIEW

File: tests/test_benchmarks.py [hunk 1]
```diff
@@ -31,30 +38,51 @@ def test_lut(self):
...
+            print("\t\t...'grid_proj'...")
+            self.assertEqual(b_lut.grid_proj, t_lut.grid_proj)
+            print("\t\t...'coord_proj'...")
+            self.assertEqual(b_lut.coord_proj, t_lut.coord_proj)
+            print("\t\t...'ll_corner'...")
+            self.assertTrue(np.isclose(b_lut.ll_corner, t_lut.ll_corner).all())
+            print("\t\t...'ur_corner'...")
+            self.assertTrue(np.isclose(b_lut.ur_corner, t_lut.ur_corner).all())
+            print("\t\t...'node_spacing'...")
+            self.assertTrue(np.equal(b_lut.node_spacing, t_lut.node_spacing).all())
```
Comment (hemmelig) [inline tests/test_benchmarks.py:74]:
```suggestion
             self.assertTrue(np.equal(b_lut.node_spacing,
                                      t_lut.node_spacing).all())
```

**Comment by hemmelig** [comment]
Last thing to add is the Changelog.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+      os: osx
+      language: generic
+      env: PYTHON_VERSION=3.8
+
+sudo: false
+
+before_install:
+  # Make Python scripts executable
+  - chmod +x tests/*
+  - chmod +x examples/Icequake_Iceland/*.py
```
Comment (TomWinder) [inline .travis.yml:40]: Necessary? Presumably not if the dike example works without this

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+  - conda config --set always_yes yes --set changeps1 no
+  - conda config --add channels conda-forge
+  - conda update -q conda
+  - conda info -a
+  - echo $PYTHON_VERSION
+  - conda create -q -n test-environment python=$PYTHON_VERSION coverage flake8 matplotlib mock numpy obspy pandas pyproj scipy
+  - source activate test-environment
+
+  # Enforce single-threaded just to stay in memory
+  - export OMP_NUM_THREADS=1;
```
Comment (TomWinder) [inline .travis.yml:65]: I think this does nothing: these variables are already set in each example script that is run, and will be over-ridden by the `scan.threads` specified in the run script.

File: tests/test_import.py [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+class TestImport(unittest.TestCase):
+    def test_import(self):
+        i = 0
+        import sys
+        if sys.version_info.major != 3:
+            print("QuakeMigrate does not support Python 2.x")
+            i += 1
+        if sys.version_info.minor < 6:
+            print("QuakeMigrate only supports Python 3.5 and up.")
```
Comment (TomWinder) [inline tests/test_import.py:25]:
```suggestion
            print("QuakeMigrate only supports Python 3.6 and up.")
```

**Review by TomWinder — CHANGES_REQUESTED** [review]
Overall looks good. Strictly it should be "2020-2021" (no spaces) which bugs me but obviously doesn't actually matter. Exciting to have CI up and running.

Request changes is purely for python version printout in test_import.py

File: .travis.yml (hunk 2 above)
Comment (TomWinder) [reply .travis.yml:40]: Nope, they're not executed. It's just python that needs permissions - so I'm surprised if `python tests/test_benchmarks.py` doesn't work without this being set.

**Comment by TomWinder** [comment]
> Last thing to add is the Changelog.

Will include this in a following PR.

**Review by TomWinder — APPROVED** [review]
Approved

**Pull request merged**